src/machine/board_pca10031.go

```diff
@@ -0,0 +1,31 @@
+// +build nrf51,pca10031
+
+// pca10031 is a nrf51 based dongle, intended for use in wireless applications.
+//
+// https://infocenter.nordicsemi.com/pdf/nRF51_Dongle_UG_v1.0.pdf
+package machine
+
+// The pca10031 has a 32kHz crystal on board.
+const HasLowFrequencyCrystal = true
+
+// LED on the pca10031
+const (
+	LED       = LED_RED
+	LED1      = LED_RED
+	LED2      = LED_GREEN
+	LED_RED   = 21
+	LED_GREEN = 22
+	LED_BLUE  = 23
```

Comment:
You could add a `LED3 = LED_BLUE` here, for completeness.

Reply:
Done.